cmd/internal/encryption/encryption_test.go

```diff
@@ -0,0 +1,56 @@
+package encryption
+
+import (
+	"log/slog"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestEncrypter(t *testing.T) {
+	// Key too short
+	_, err := New(slog.Default(), "tooshortkey")
+	require.EqualError(t, err, "key length:11 invalid, must be 16,24 or 32 bytes")
+
+	// Key too short
+	_, err = New(slog.Default(), "19bytesofencryption")
+	require.EqualError(t, err, "key length:19 invalid, must be 16,24 or 32 bytes")
+
+	// Key too long
+	_, err = New(slog.Default(), "tooloooonoooooooooooooooooooooooooooongkey")
+	require.EqualError(t, err, "key length:42 invalid, must be 16,24 or 32 bytes")
+
+	e, err := New(slog.Default(), "01234567891234560123456789123456")
+	require.NoError(t, err, "")
+
+	input, err := os.CreateTemp("", "encrypt")
```

Comment:
It would be nice if we could utilize the `afero` filesystem abstraction for this package, too. This way we could execute the tests in memory and we do not depend on the host filesystem for test execution. It's already used in many places in this project.